recipe/meta.yaml

```
@@ -25,7 +25,7 @@ source:
     - fix_msvc_build_issue.diff
 
 build:
-  number: 1
+  number: 2
```

Is there any need to bump the build number?
```suggestion
  number: 1
```